lib/ios/RNNBottomTabsOptions.h

```diff
@@ -18,5 +18,6 @@
 @property (nonatomic, strong) Text* barStyle;
 @property (nonatomic, strong) Text* fontFamily;
 @property (nonatomic, strong) Text* titleDisplayMode;
+@property (nonatomic, strong) Text* tabsAttachMode;
```

Comment:
We resolve these option later on, we can't use enum here because our options mechanism rely on objects (merge, withDefault). We can use a subclass of `Text` object that will return enum value, wdyt?